### ISSUE TITLE
RFE: Don't scroll to the top in dictionary manager when a download finishes

### ISSUE THREAD

**Issue by genodeftest** [rdoeffinger/Dictionary#151]
When downloading multiple dictionaries, after each download/installation, the download manager will scroll to the top, interrupting user experience.

## Steps to reproduce:
1. Open QuickDic
2. Open its _Dictionary Manager_
3. Scroll down a bit
4. Under _Downloadable dictionaries_, download any dictionary
5. Keep scrolling

## What happens:
As soon as any download (or installation) finishes, the view will jump to the top. To continue where he left off, the user has to scroll down again. This is especially cumbersome when not using search to filter text so the scrollable length of the view is very large

## What should happen:
Don't jump to the top (or anywhere) when download/installation finish.

## Additional info:
I'm using QuickDic 5.6.2 installed from F-Droid on Android 11.

**Comment by Aphexus** [comment]
I agree with this. Very annoying when trying to install multiple dictionaries as it will keep jumping back on a fast connection.
